addons/website/__manifest__.py

```diff
@@ -168,6 +169,7 @@
             'website/static/src/js/theme_preview_kanban.js',
             'website/static/src/js/theme_preview_form.js',
             'website/static/src/client_actions/*/*.js',
+            ('remove', 'website/static/src/client_actions/website_preview/website_preview_test_mode.js'),
```

Comment:
Mouarf, you have to remove it because it is included by the glob pattern just above? I don't think this is a good usecase for the "remove" feature. At worst can't this be put in another folder? Or at best maybe the glob pattern can be written with exclusions (no clue)

Reply:
That's literally 99% of this `remove` use case in our codebase (including a whole folder and removing one or a few files).
I don't mind changing it if this PR was to be going further (see discussion in another review point).

Reply:
Mouais, that's probably marseillais since in the first 10 occurences I saw it is more like 20% than 99% but ok, I did not say it is wrong anyway :wink: But I would never do that myself in any case :shrug:

Reply:
23 out of the 43 occurrences actually, indeed not 99%, I clicked on 4-5 and they were all like that so I didn't went further :yum: 
Not a big deal anyway indeed, I can change it if needed as I said :+1: